code/common/compress.q

```diff
@@ -173,8 +173,8 @@ compress:{[filetoCompress;algo;blocksize;level;sizeuncomp]
                 $[not 0=algo;statstab ,: (filetoCompress;algo;(-21!filetoCompress)`compressedLength;sizeuncomp);statstab ,: (filetoCompress;algo;comprL;sizeuncomp)]];
```

Comment:
Pull `statstab,:` out in front of the conditional then it only needs written once. The lists in each condition are also repetitive, could be room for further trimming.